nixos/modules/security/acme.nix

```diff
@@ -87,13 +87,13 @@ let
         default = {};
         example = literalExample ''
           {
-            "example.org" = "/srv/http/nginx";
+            "example.org" = null;
             "mydomain.org" = null;
           }
         '';
         description = ''
-          A list of extra domain names, which are included in the one certificate to be issued, with their
-          own server roots if needed.
+          A list of extra domain names, which are included in the one certificate to be issued.
+          Setting a distinct server root is deprecated and not functional in 20.03+
```

Comment:
This might be off-topic for this PR, but on top of this message, could we generate a eval-time warning using the assertion module https://github.com/NixOS/nixpkgs/blob/b67bc34d4e3de1e89b8bb7cd6e375ba44f1ae8ca/nixos/modules/misc/assertions.nix#L21 if `acmeRoot!=null` here?

Do we plan to backport this commit to 19.09? If not, I think this is safe to assume this option is deprecated and not functional, hence dropping the `in 20.03+` part of this message.

Reply:
Also, maybe it's also time to remove the acme root from this example: https://github.com/NixOS/nixpkgs/blob/c9f6e5f161f64701bce0527c80628eb637eae0ac/nixos/modules/security/acme.nix#L253

Reply:
So I actually tried doing that back when I did the DNS-01 update, but I couldn't figure out how to add a deprecation warning to a submodule (which is what the certs are). When I rewrite the whole module I'm actually going to add a warning and a rewrite rule to change them to a list.